core.css

```diff
@@ -109,5 +140,57 @@ p {
 .p-lg {
   font-size: 1.6rem;
 }
-@media screen and (max-width: 768px) {
+.text_line_th {
+  text-decoration: line-through;
 }
+.text_uc {
+  text-transform: uppercase;
+}
+.text_cl {
+  text-transform: capitalize;
+}
+.comp_snippet {
+  margin-top: 4rem;
+  overflow: auto;
+  border-radius: 12px;
+  background-color: #011627;
+  text-align: center;
+  width: 100% !important;
+  font-size: 1.6rem !important;
+}
+
+:not(pre) > code[class*="language-"],
+pre[class*="language-"] {
+  background: inherit !important;
+}
+.token.tag {
+  color: #569cd6 !important;
+}
+.toolbar > .toolbar > .toolbar-item > button:hover,
+div.code-toolbar > .toolbar > .toolbar-item > span:focus,
+div.code-toolbar > .toolbar > .toolbar-item > span:hover {
+  color: var(--white) !important;
+  text-decoration: none;
+  cursor: pointer !important;
+}
+/* .toolbar {
+  position: sticky !important;
+} */
+.copy-to-clipboard-button,
+.toolbar,
+.toolbar-item {
+  background: 0 0;
+  border: 0;
+  font: inherit;
+  font-size: 1.6rem !important;
```

Comment:
avoid using important

Reply:
> avoid using important

I'm overriding prismjs CSS, without !important, it won't work. let me know if you have a solution for that.